promcda/utils/utils_for_main.py

```diff
@@ -1055,16 +819,65 @@ def run_mcda_with_indicator_uncertainty(input_config: dict, input_matrix: pd.Dat
                           iterative_random_w_score_means=None,
                           iterative_random_w_score_stds=None,
                           iterative_random_w_score_means_normalized=None,
-                          input_matrix=input_matrix, config=input_config,
+                          input_matrix=input_matrix, config=extracted_values,
                           is_robustness_indicators=is_robustness_indicators)
 
 
-def _compute_scores_for_all_random_weights(indicators: dict, is_sensitivity: str,
-                                           weights: Union[List[str], List[pd.DataFrame], dict, None],
-                                           f_agg: str) -> tuple[Any, Any, Any, Any]:
+def check_input_matrix(input_matrix: pd.DataFrame) -> pd.DataFrame:
+    """
+    Check the input matrix for duplicated rows in the alternatives column, rescale negative indicator values
+    and drop the index column of alternatives.
+
+    Parameters:
+    - input_matrix: The input matrix containing the alternatives and indicators.
+
+    Raises:
+    - ValueError: If duplicated rows are found in the alternative column.
+    - UserStoppedInfo: If the user chooses to stop when duplicates are found.
+
+     :param input_matrix: pd.DataFrame
+     :rtype: pd.DataFrame
+     :return: input_matrix
+    """
+    if input_matrix.duplicated().any():
+        raise ValueError('Error: Duplicated rows in the alternatives column.')
+    elif input_matrix.iloc[:, 0].duplicated().any():
+        logger.info('Duplicated rows in the alternatives column.')
```

Comment:
Both the messages in `if..else..` condition are confusing and appears same even though the condition checks are different. 
From what I understand `input_matrix.duplicated().any()` checks for entire duplicated row but `elseif` part checks for duplicated value in just first column marked with index `0`. So basically this turns out that you need to log information when there are duplicates in just first column and raise error when one or more than one entire row is duplicated. 

Would be great to edit these messages accordingly. 

Would also be great to log the impact in `elif` condition because despite of duplicate row in first column you choose to continue.